docs/blog/2019-06-21-dropping-support-for-node-6/index.md

```diff
@@ -0,0 +1,94 @@
+---
+title: "Dropping Support for Node 6"
+date: 2019-06-21
+author: Dustin Schau
+excerpt: "We're dropping support for Node 6 in minor releases of core Gatsby packages. Learn more about why this decision was made."
+tags:
+  - core
+  - deprecation
+---
+
+Effective in Gatsby v2.10.0, we are making a potentially breaking change and dropping support for Node 6. We don't take this decision lightly, and realize that there is _potential_ for breaking applications by not following [semantic versioning][semver] _strictly_ and releasing Gatsby v3.0.0. However, we feel that this decision is in the best interest of our users and enables the Gatsby Core team to focus more efficiently on delivering fixes and features. In this post, I outline the reasoning for why we're making this decision and how it potentially impacts you, a Gatsby user.
+
+## Why
+
+Several months ago (01/03/2019) we opened a [Request for Commentary (RFC)][rfc], outlining our strategy for dropping Node 6 and making this breaking change. We began by making documentation changes to gently guide Gatsby users towards making the change to Node 8 or greater. We did this in several ways, specifically:
+
+1. Used `async`, `await`, and other non-supported features in Node 6 in our documentation
+   - Copying code directly from the docs will manifest a runtime error in Node 6, and will gently guide towards updating to Node 8 or greater
+1. [Documented that our minimum supported version is Node 8 or greater][node-8-docs]
+
+These approaches jointly served as a gentle guide towards encouraging users to upgrade to Node 8 _prior to this breaking change._ However, up to this point, I haven't defended why we're making the breaking change. Let me do just that.
+
+## Why a Breaking Change?
+
+We don't take lightly the potential for breaking changes. Our ultimate goal in publishing versions of Gatsby and plugins is that we _never_ want to make a breaking change without a major version bump. In fact, our approach is to publish early and often so that we can get new features and fixes in user's hands as soon as possible. Our intent is that users will update their Gatsby dependencies regularly and often to get these new features and fixes as quickly as possible. This model crucially depends upon trust in being able to upgrade these dependencies as often as we ship them.
+
+However, we have regularly and repeatedly found it extremely challenging to keep delivering these valuable features and fixes while still maintaining support for Node 6. Many of our dependencies (and further still, dependencies of our dependencies) have dropped support for Node 6 in minor releases, thereby preventing Gatsby from using these dependencies to get needed features and fixes.
```

Comment:
The examples I provide are examples of this! e.g. the skipping unit tests in Node 6 because remark-mdx dropped support (dependency of gatsby-remark-responsive-iframe). Perhaps I can spell this out more clearly?